batch7crud-roll1/src/main/webapp/resources/css/style.css

```diff
@@ -0,0 +1,236 @@
+body{
+    background-color: #E2E2E2;
+    font-family: Arial;
+}
+
+a{
+    text-decoration: none;
+}
+
+.company-name h2{
+    color: #FFFFFF;
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.header-menu{
+    float: right;
+    padding-top: 10px;
+}
+
+.header-menu ul{
+
+}
+
+.header-menu ul li{
+}
+
+.header-menu ul li a{
+    display: block;
+    padding: 10px;
+    color: #FFF;
+    background-color: #357ebd;
+    border-radius: 4px;
+}
+
+.navigation ul{
+
+}
+
+.navigation ul li{
+    border-bottom: 1px solid #444444;
+}
+
+.navigation ul li a{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Simplify this over-specified selector [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aoverspecific-selectors)